### DISCUSSION TITLE
Way to go guys

### DISCUSSION THREAD

**Discussion by saudiwin** [surveydown-dev/surveydown#159] [Feature Requests]
### Description

This isn't actually a feature request. Just wanted to say I'm super impressed with this package, its development, and the impact it will have on the wider research community. Am preparing now for its first deployment to survey research in Africa and the Middle East. Will post when I have findings!!

**Comment by jhelvy** [comment]
Thanks for sharing! We actually have a [post in the discussion](https://github.com/orgs/surveydown-dev/discussions/144) for comments exactly like these - would you mind posting this there?

Also, please do install the latest version (v0.7.2) before fielding. We made some changes since our last release that improve the session restoration (i.e. if a user closes or refreshes their browser, their progress will be restored) and loading speed. Run it once locally to render the survey content, then all future runs will launch from the pre-rendered content, which is super fast.

**Reply by saudiwin** [reply]
sure thing, I'll cross-post
